sormas-api/src/main/java/de/symeda/sormas/api/infrastructure/InfrastructureSyncDto.java

```diff
@@ -20,6 +19,7 @@
 import de.symeda.sormas.api.user.UserDto;
 import de.symeda.sormas.api.user.UserRoleDto;
 
+@AuditedClass(includeAllFields = true)
```

Comment:
I don't think the content of this is worth being audited. It's just access to infrastructure data and it may create a big amount of log data.